### DISCUSSION TITLE
gamePort does not properly open or map as configured

### DISCUSSION THREAD

**Discussion by darkjeti** [PR3SIDENT/enshrouded-server#27] [Q&A]
When running the gameserver as a service, the query port seems to open properly(default 15637) however the game port does not open (default 15636). Instead there are other random ports:

```

steam@enshrouded:~/enshrouded/logs$ ss -tulpn | grep -E 'wine|shroud'
Netid        State         Recv-Q         Send-Q                 Local Address:Port                  Peer Address:Port        Process      
udp          UNCONN        0              0                            0.0.0.0:46734                      0.0.0.0:*            users:(("enshrouded_serv",pid=9147,fd=279),("wineserver",pid=9124,fd=383))        
udp          UNCONN        0              0                            0.0.0.0:15637                      0.0.0.0:*            users:(("enshrouded_serv",pid=9147,fd=235),("wineserver",pid=9124,fd=354))        
tcp          LISTEN        0              4096                       127.0.0.1:36315                      0.0.0.0:*            users:(("wineserver",pid=9124,fd=331))                                            

root@enshrouded:~# ss -tulpn | grep -E 'wine|shroud'
udp   UNCONN 0      0            0.0.0.0:45817      0.0.0.0:*    users:(("enshrouded_serv",pid=9279,fd=274),("wineserver",pid=9256,fd=383))
udp   UNCONN 0      0            0.0.0.0:15637      0.0.0.0:*    users:(("wineserver",pid=9256,fd=375))                                    
tcp   LISTEN 0      4096       127.0.0.1:44997      0.0.0.0:*    users:(("wineserver",pid=9256,fd=330))   


steam@enshrouded:~/enshrouded$ cat enshrouded_server.json 
{

    "name": "test123",

    "password": "testing123",

    "saveDirectory": "./savegame",

    "logDirectory": "./logs",

    "ip": "0.0.0.0",

    "gamePort": 15636,

    "queryPort": 15637,

    "slotCount": 8

}

```

Installed via InstallServer.sh script on Debian 12.

**Comment by mabo89** [comment]
Same issue here: 
Fresh install of Ubuntu 22.04

Does not matter if the server is either run as a service or directly called with wine64. 

Changing the game and query port in enshrouded_server.json does not help either. 
Query port is changed correctly on next startup, but gameport is just a random port as it seems.

udp   UNCONN 0      0                 0.0.0.0:33921      0.0.0.0:*    users:(("enshrouded_serv",pid=43615,fd=282),("wineserver",pid=43556,fd=484))
udp   UNCONN 0      0                 0.0.0.0:22501      0.0.0.0:*    users:(("wineserver",pid=43556,fd=453))
tcp   LISTEN 0      4096            127.0.0.1:40577      0.0.0.0:*    users:(("wineserver",pid=43556,fd=428))

Also tried to alternate the IP between 0.0.0.0, 127.0.0.1 and the actual public IP of the server.
